shopfloor_batch_automatic_creation/actions/picking_batch_auto_create.py

```diff
@@ -104,11 +104,14 @@ def _search_pickings(self, picking_types, user=None, commercial_partner=None):
         # We can't use a limit in the SQL search because the 'priority' fields
         # is sometimes empty (it seems the inverse StockPicking.priority field
         # mess up with default on stock.move), we have to sort in Python.
```

Comment:
This should be dropped according to the last changes. Can't find such issue in v14 but have a check on your DBs.